readme.md

```diff
@@ -714,7 +714,7 @@ The prompt will list options based on user input. Type to filter the list.
 Use <kbd>⇧</kbd>/<kbd>⇩</kbd> to navigate. Use <kbd>tab</kbd> to cycle the result. Use <kbd>Page Up</kbd>/<kbd>Page Down</kbd> (on Mac: <kbd>fn</kbd> + <kbd>⇧</kbd> / <kbd>⇩</kbd>) to change page. Hit <kbd>enter</kbd> to select the highlighted item below the prompt.
 
 The default suggests function is sorting based on the `title` property of the choices.
-You can overwrite how choices are being filtered by passing your own suggest function.
+You can overwrite how choices are being filtered bypassing your own suggest function.
```

Comment:
"passing" is a fixed termin in computer sciense. Please read on at https://www.mathwarehouse.com/programming/passing-by-value-vs-by-reference-visual-explanation.php

so, this change is invalid.

Reply:
Please revert change